priv/repo/migrations/20190211103356_create_capitals_distances.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Co2Offset.Repo.Migrations.CreateCapitalsDistances do
+  use Ecto.Migration
+
+  def change do
+    create table(:capitals_distances) do
+      add :capital_a, :string
+      add :capital_b, :string
+      add :distance, :float
```

Comment:
Please add index for distance.
Primary use of the table will be finding the rows with value closes to distance column

Reply:
added